crdb/tx.go

```diff
@@ -50,6 +50,17 @@ func ExecuteTx(db *sql.DB, fn func(*sql.Tx) error) (err error) {
 	if err != nil {
 		return err
 	}
+	return ExecuteInTx(tx, func() error { return fn(tx) })
+}
+
+type Tx interface {
+	Exec(query string, args ...interface{}) (sql.Result, error)
+	Commit() error
+	Rollback() error
+}
+
+// ExecuteInTx runs fn inside tx which should already have begun.
```

Comment:
I looked at the sql.Tx and didn't see any way to test for statements already being executed.  I can't think of any way to enforce this, I'm afraid the comment is going to have to do...

Any suggestions on how to phrase this?